translator/shared.py

```diff
@@ -1,11 +1,12 @@
-"""This module provides a location for code that we want to share between all translators."""
+"""Provide a location for code that we want to share between all translators."""
 
 from exceptions import ASNError
 
```

Comment:
Also why variablize this?

Reply:
This (and several others) were failing a check that a doc string not start with "This": https://docs.astral.sh/ruff/rules/docstring-starts-with-this/

Reply:
Sorry my aim was off on the line. This queywad about the `MAX_ASN_VAL` part